mods/ra/weapons/missiles.yaml

```diff
@@ -90,16 +89,15 @@ HellfireAG:
 		Inaccuracy: 128
 		Image: DRAGON
 		HorizontalRateOfTurn: 10
-		RangeLimit: 4c819
+		RangeLimit: 8c819
```

Comment:
@GraionDilach Ah ok that makes much more sense. That said, 4 x 1.2 = 4.8 exactly. What maths am I missing to get the 4c8**19** part. (I assume c simply stands for cells)

Also I assume I can mix up the percentages if need be? I.e. longbow max range is 1.7 normal range while mig max range is 1.4 normal range (example percentages)